### ISSUE TITLE
MQTT en_01 / en_02 incorrect (v9_8)

### ISSUE THREAD

**Issue by Superbert25** [patience4711/read-APSystems-YC600-QS1-DS3#99]
This issue is related to version 9_8:

MQTT reports faulty values for en_01 and en_02. Screenshot ECU server:
![IMG_0875](https://user-images.githubusercontent.com/94918025/230155872-c3c5f7a2-85b9-48d6-900e-2ada973981af.jpeg)

Screenshot from my automation server:
![IMG_0874](https://user-images.githubusercontent.com/94918025/230155984-0d7224a9-8ddd-4644-a421-11fc4d7c8766.jpeg)

Power values are matching, energy is completely wrong (was earlier today fine). Could have to do with exceeding some value?

R.

Eeg.

**Comment by Superbert25** [comment]
After a reboot of the ECU, the webserver was also displaying the faulty values...maybe the inverters have rebooted?
<img width="303" alt="image" src="https://user-images.githubusercontent.com/94918025/230157248-45f7c9b7-6e8a-4c3c-b396-9742c1f51062.png">
I'm puzzled....

**Comment by patience4711** [comment]
Most likely this is an coincident. The only way this can happen is that both ESP and inverter resetted. When you installed the software, the esp resetted. So indeed it looks like the inverters did so too. Maybe due to a grid failure? Or was this at the end of the day?

Anyway i trust that it works oke now?

**Comment by Superbert25** [comment]
Time will tell. 

It was indeed at the end of the day…but there was still sufficient light for a bit of power production.

I’ll keep monitoring and report back here

**Comment by patience4711** [comment]
@Superbert25   You seem to use the cc2530/2591 zigbee module, can you tell which firmware you flashed on it?

**Comment by swbouman** [comment]
@patience4711 I’m also using the cc2530+2591 setup. I flashed the 2530+2591 firmware, available by koenk. 

I had to convert the .hex file to a flashable bin file. 
the 2530+2591 setup is running on standard 2530 firmware, but the amplifier is. It activated at that moment. 

Right now I achieved a range of 20 meters and the device is mounted in house, below the ceiling. 

Although I have to aim the antenna better hence the roof is reflecting the signals when the sun is on a certain angle.
